Gemfile

```diff
@@ -99,7 +99,7 @@ gem 'rails-i18n', '~> 7.0.8' # Localizations for Rails built-ins
 gem 'redcarpet', '~> 3.5' # Process markdown in form textareas (justifications)
 gem 'sass-rails', '~> 5.1', require: false # For .scss files (CSS extension)
 gem 'scout_apm' # Monitor for memory leaks
-gem 'secure_headers', '~> 6.3' # Add hardening measures to HTTP headers
+gem 'secure_headers', '~> 7' # Add hardening measures to HTTP headers
```

Comment:
Ok. This is a new major number, but `secure_headers` gem's CHANGELOG doesn't even discuss *what* changed in the 7 series, nor what steps you should take (if any) when upgrading). Not *your* fault, I'm just grumbling about bad documentation :-).

Reply:
Should have some minor performance improvements